include/caf/net_state.rst

```diff
@@ -0,0 +1,38 @@
+.. _caf_net_state:
+
+CAF: Network state
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |net_state| of the :ref:`lib_caf` (CAF) is responsible for broadcasting the events about current network state.
+The module provides different backends for available networks (like LTE or OpenThread).
+
+Configuration
+*************
+
+You can enable |net_state| by selecting the :option:`CONFIG_CAF_NET_STATE` option in the configuration.
```

Comment:
We've now changed :option: to :kconfig: due to Zephyr upmerge changes. So please replace all :option: with :kconfig: .

Reply:
ok